### DISCUSSION TITLE
Writing an array of UDTs

### DISCUSSION THREAD

**Discussion by larson-7** [ottowayi/pycomm3#206] [Q&A] [answered]
Hello,

Unless I'm utilizing the incorrect syntax, it looks like trying to write an array of UDTs is not supported. I can read the said arrays of UDTs and they are a list of dictionaries ~ [{UDT}, {UDT}, ... , {UDT}] . 

Trying to write this same setup I get an error: Invalid Tag Request - RequestError('Unable to create a writable value'). 

Is there something i'm missing?

**Comment by ottowayi** [comment, marked as answer]
Writing arrays of UDTs is definitely supported, so probably just need to make a some tweaks to what you're doing.  Can you share some some of your code and the structure of the UDT?  Also can you enable [logging](https://docs.pycomm3.dev/en/latest/getting_started.html#logging)?  That will help with figuring out what the issue is.

**Reply by larson-7** [reply]
The logging feature was immensely helpful, thank you for your suggestion. I needed to RTFM. It was all on me with an improper UDT member ... spelling will get you. 

Thank you for responding back in a timely manner, your help, and most importantly developing and managing Pycomm3!

**Reply by ottowayi** [reply]
Hey no problem, glad you got it figured out!
